library/src/main/scala/scala/scalajs/runtime/LinkingInfo.scala

```diff
@@ -10,6 +10,9 @@ trait LinkingInfo extends js.Object {
 
   /** Whether we are assuming ECMAScript 6 support or not. */
   val assumingES6: Boolean = js.native
+
+  /** Version of the linker */
+  val linkerVersion: String = js.native
```

Comment:
Actually, I just realized this must be a `js.UndefOr[String]`. If we link a library 0.6.8 with a linker 0.6.7 (which is possible as long as 0.6.8 remains forward compatible), the `linkerVersion` will not be there yet. `loadSystemProperties()` should only populate `java.vm.version` if this field is not undefined.